src/gc-tls.h

```diff
@@ -49,7 +49,7 @@ typedef struct {
 
 typedef struct {
```

Comment:
Some comments on each field for documentation would be so good.

Reply:
While I agree with the suggestion, will leave it to a future PR.